src/scripts/create_machine.sh

```diff
@@ -22,6 +22,19 @@ vmExists(){
    virsh -c "$CONNECTION_URI" list --all | awk  '{print $2}' | grep -q --line-regexp --fixed-strings "$1"
 }
 
+readQemuConf(){
+    IFS="="
+    while read -r name value
+    do
+	# removes whitespaces
+	name="$(echo -e "${name}" | tr -d '[:space:]')"
+	if [ "$name" = "$1" ]; then
+		# removes " and whitespaces
+		echo -e "${value//\"/}" | tr -d '[:space:]'
```

Comment:
You don't normally want to remove whitespace *inside* the value, especially not when it's quoted. How about something like this:

    awk '/^[[:space:]]*'"$1"'[[:space:]]*=/ { sub(/^.*= "?/, ""); sub(/"[[:space:]]*$/, ""); print }' /etc/libvirt/qemu.conf

I know it's a mouthful, but I believe it's quite a bit more precise. It also avoids the side effect of setting `$IFS`, which will influence the code afterwards (which can  be very surprising and hard to see)